Feature update

ScribbleTAS opened the pull request:
# Changes
- Removed command parser and added CommandHandler, which dynamically creates commands instead of reading them from a file
- Renamed package de.pfannekuchen to com.minecrafttas
- Added better logging output
- Improved reactionroles, which I accidentally haven't committed before

ScribbleTAS committed:
[WIP] Removed command parser

ScribbleTAS committed:
Added custom commands editable within discord

- /customcommand upsert will create or update a command
- /customcommand rename will rename an existing command
- /customcommand remove will remove a command

- Added better logging
- Removed reaction role commands for now

ScribbleTAS committed:
Renamed package to com.minecrafttas

ScribbleTAS committed:
Skeleton for reaction roles

ScribbleTAS committed:
Add reactionsroles

ScribbleTAS committed:
Added logging messages

PancakeTAS commented:
recently learned that LOGGER.warn/error have a method that accepts a throwable/exception. If you use that one you apparently don't need to put {}

ScribbleTAS commented:
Note to self: Remove emoji java
Also documentation, jeez

ScribbleTAS committed:
More logging messages

ScribbleTAS committed:
Remove admin gate

ScribbleTAS committed:
Small additions to features

- Fixed events being already acknowledged and throwing errors
- Added the ability to add reactionroles to non bot messages
- Added the ability to add non embed messages to custom commands (similar to tascompbot)
- Changed storage folder name of reactionroles from reactionsrole to reactionrole
